src/Directives.jl

```diff
@@ -95,11 +99,20 @@ function parsebracket end
 parsebracket(text :: AbstractString) = parsebracket(getdirective(text)...)
 parsebracket{D}(:: Directive{D}, text) = error("Unknown directive: '$D'")
 
+function parsebracket{D}(TD :: Directive{D}, arg, text)
+    if strip(arg) == ""
+        parsebracket(TD, text)
+    else
+        error("parsebracket not defined for directive '$D' with positional argument.")
+    end
+end
+
 function parsebracket(:: directive"docs", text)
+    eval(Expr(:toplevel, Expr(:using, :Lexicon)))
```

Comment:
If it's ok, I might take a back-seat for a bit - it sounds like you have a good way to do it (which I don't understand)! I can help, assuming I'm helping (!), more once something is working a bit? (I'll keep playing.)

Moving features to Docile makes a lot of sense, I think this is a good way forward. The distinction was becoming a little unclear, especially with directives.

FYI "unpublished"/WIP slides are here http://hayd.github.io/s/juliadocs/#1, but it's not for another month. Hopefully it'll be some good ~~Lexicon~~ Docile advertisement :)